content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -249,6 +249,17 @@ these are:
   waiting for a fast one; if all servers are busy, the algorithm falls back to the `sed`
   behavior.
 
+* `mh` (Maglev Hashing): Assigns incoming jobs based on
+  [Google's Maglev hashing algorithm](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/44824.pdf),
+  This scheduler has two flags: `mh-fallback`, which enables fallback to a different
+  server if the selected server is unavailable, and `mh-port`, which adds the source port number to
+  the hash computation. When using `mh`, IPVS always uses the `mh-port` flag.
```

Comment:
I missed this: "IPVS always uses" here should rather be "kube-proxy always sets". And perhaps mention that mh-fallback isn't used

Reply:
When using `mh`, kube-proxy always sets the `mh-port` flag and does not enable the `mh-fallback` flag.